.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 14
+
+# Issues or Pull Requests with these labels will never be considered stale
+exemptLabels:
+  - "enhancement"
+
+# Label to use when marking as stale
+staleLabel: "awaiting input"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
```

Comment:
I think it's meaningless to have a comment that says "label was removed", since it will be seen after any comment appears in the issue. This is just my opinion, though.